packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -4034,8 +4053,13 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
 
         mesh.checkCollisions = parsedMesh.checkCollisions;
 
+        // Kept for backwards compatibility
         if (parsedMesh.overrideMaterialSideOrientation !== undefined) {
-            mesh.overrideMaterialSideOrientation = parsedMesh.overrideMaterialSideOrientation;
+            mesh.sideOrientation = parsedMesh.overrideMaterialSideOrientation;
+        }
+
+        if (parsedMesh.materialSideOrientation !== undefined) {
+            mesh.sideOrientation = parsedMesh.overrideMaterialSideOrientation;

```

Comment:
Should be `parsedMesh.materialSideOrientation` instead of `parsedMesh.overrideMaterialSideOrientation`.